<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormServoParamButton</class>
 <widget class="QWidget" name="FormServoParamButton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="labelServoParam">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 14px;
font-weight: 350;
line-height: 40px;
color: #2D3440;</string>
     </property>
     <property name="text">
      <string>伺服参数</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnCheckServoParam">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 3px;
opacity: 1;
background: #FAFDFF;
border: 1px solid #DFE3E5;
color: #2D3440;
font-size: 14px;
font-weight: 350;
    padding-left:6px;
    padding-right:6px;</string>
       </property>
       <property name="text">
        <string>检查关键伺服参数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUpdateServoParam">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 3px;
opacity: 1;
background: #0047BB;
font-size: 14px;
font-weight: 350;
line-height: normal;
text-align: center;
align-items: center;
color: #FAFDFF;
    padding-left:6px;
    padding-right:6px;</string>
       </property>
       <property name="text">
        <string>覆盖最新参数</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutInfo">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="labelServoState">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">.QLabel{
	opacity: 0.8;
	font-size: 14px;
	font-weight: 350;
	line-height: 40px;
	text-align: center;
	color: #5E626E;
}
.QLabel[enabled=true]{
	opacity: 0.8;
	font-size: 14px;
	font-weight: 350;
	line-height: 40px;
	text-align: center;
	color: #0047BB;
}</string>
          </property>
          <property name="text">
           <string>需要升级</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="labelServoStateInfo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 12px;
font-weight: 350;
color: #F21434;
background-image: url(:/image/images/warnicon.png);
background-position: left center;
background-repeat:no-repeat;
padding-left:30px;</string>
          </property>
          <property name="text">
           <string>请先一键升级完固件后，重启控制柜，再次连接成功后，手动点击覆盖按钮</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutCheck" stretch="1,2">
        <property name="spacing">
         <number>10</number>
        </property>
        <item alignment="Qt::AlignRight|Qt::AlignVCenter">
         <widget class="QLabel" name="labelDoing">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>80</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border:none;</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
         <widget class="QLabel" name="labelCheck">
          <property name="styleSheet">
           <string notr="true">.QLabel{
	opacity: 0.8;
	font-size: 14px;
	font-weight: 350;
	line-height: 40px;
	text-align: center;
	color: #0047BB;
}</string>
          </property>
          <property name="text">
           <string>检查关键伺服参数中，大概耗时1分钟</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
